### ISSUE TITLE
FR: Add min/max/avg requests / second in last hour/day/period

### ISSUE THREAD

**Issue by SjonHortensius** [allinurl/goaccess#599]
I'd like to see the min/max/average number of requests for a few periods such as previous hour, day or entire period.

**Comment by allinurl** [comment]
The min/max is already in the web interface. I can certainly add the average though. I expect to add these three soon in the terminal output as well.

As far as filtering by a period of time, it will definitely be addressed on issue #117.

**Comment by SjonHortensius** [comment]
I can only find the min/max per period; not for the last day or hour. Am I missing something? Especially with the real-time view it would be nice to see average requests / second displayed which I currently cannot find

**Comment by allinurl** [comment]
That's right, you can't filter right now by a specific period of time such as last day/hour. This will be added when #117 is implemented. I plan to add an interface that will allow users to specify what filters to apply, e.g., date/time, specific token such as UA, request, etc.

**Comment by SjonHortensius** [comment]
Ah, I see. Thanks for explaining

**Comment by allinurl** [comment]
Forgot to mention, currently you can do some preprocessing to filter out your data using pipes. Take a look at the man page for some [**examples**](https://goaccess.io/man#examples).
